platform/sun50i/devices.c

```diff
@@ -54,3 +56,12 @@ static struct device r_intc = {
 	.drv     = &sun4i_intc_driver,
 	.drvdata = SUN4I_INTC_DRVDATA { { 0 } },
 };
+
+static struct device r_pio = {
+	.name     = "r_pio",
+	.clock    = CCU_GATE(R_CCU_GATE_R_PIO) | CCU_RESET(R_CCU_RESET_R_PIO),
```

Comment:
There's no reset in the hardware, so just omit the reset part from the clock description.